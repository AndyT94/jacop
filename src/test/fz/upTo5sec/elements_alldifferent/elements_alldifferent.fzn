array [1..8] of var int: element_table:: output_array([1..4,1..2]) = [1,6,2,9,3,2,4,9];
array [1..8] of var int: items:: output_array([1..4,1..2]) = [2,9,1,6,4,9,3,2];
solve  satisfy;
