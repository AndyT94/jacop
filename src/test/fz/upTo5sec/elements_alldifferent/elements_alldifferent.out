element_table = array2d(1..4, 1..2, [1, 6, 2, 9, 3, 2, 4, 9]);
items = array2d(1..4, 1..2, [2, 9, 1, 6, 4, 9, 3, 2]);
----------